ui/src/app/views/homepage/component.ts

```diff
@@ -162,9 +147,6 @@ export class HomepageComponent implements OnInit, OnDestroy {
     }
     // Don't show the banner after 1 week as their account would
     // have been disabled had they not enabled 2-factor auth.
-    if (new Date().getTime() - this.firstSignIn.getTime() > 1 * 7 * 24 * 60 * 60 * 1000) {
-      return false;
-    }
-    return true;
+    return (new Date().getTime() - this.firstSignIn.getTime() > 7 * 24 * 60 * 60 * 1000);
```

Comment:
I think this flipped the logic, so this should be not-ed or the sign should be flipped.